content/ja/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,120 @@
+description: >-
+  自動スケーリングによって、何らかのかたちでワークロードを自動的に更新できます。これによりクラスターはリソース要求の変化に対してより弾力的かつ効率的に対応できるようになります。
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Kubernetesでは、現在のリソース要求に応じてワークロードをスケールできます。
+これによりクラスターはリソース要求の変化に対してより弾力的かつ効率的に対応できるようになります。
+
+ワークロードをスケールするとき、ワークロードによって管理されるレプリカ数を増減したり、レプリカで使用可能なリソースをインプレースで調整できます。
+
+ひとつ目のアプローチは _水平スケーリング_ と呼ばれ、一方でふたつ目のアプローチは _垂直スケーリング_ と呼ばれます。
+
+ユースケースに応じて、ワークロードをスケールするには手動と自動の方法があります。
+
+<!-- body -->
+
+## ワークロードを手動でスケーリングする
+
+Kubernetesはワークロードの _手動スケーリング_ をサポートします。
+水平スケーリングは `kubectl` CLIを使用して行うことができます。
+垂直スケーリングの場合、ワークロードのリソース定義を `パッチ適用` する必要があります。
+
+両方の戦略の例については以下をご覧ください。
+
+- **水平スケーリング**: [Running multiple instances of your app](/docs/tutorials/kubernetes-basics/scale/scale-intro/)
+- **垂直スケーリング**: [Resizing CPU and memory resources assigned to containers](/docs/tasks/configure-pod-container/resize-container-resources)
+
+## ワークロードを自動でスケーリングする
+
+Kubernetesはワークロードの _自動スケーリング_ もサポートしており、これがこのページの焦点です。
+
+Kubernetesにおける _オートスケーリング_ の概念は一連のPodを管理するオブジェクト(例えば{{< glossary_tooltip text="Deployment" term_id="deployment" >}})を自動的に更新する機能を指します。
+
+### ワークロードを水平方向にスケーリングする
+
+Kubernetesにおいて、 _HorizontalPodAutoscaler_ (HPA)を使用してワークロードを水平方向に自動的にスケールできます。
+
+これはKubernetes APIリソースおよび{{< glossary_tooltip text="コントローラー" term_id="controller">}}として実装されておりCPUやメモリ使用率のような観測されたリソース使用率と一致するようにワークロードの{{<glossary_tooltip text="レプリカ" term_id="replica" >}}数を定期的に調整します。
+
+Deployment用のHorizontalPodAutoscalerを構成する[ウォークスルーチュートリアル](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough)があります。
+
+### ワークロードを垂直方向にスケーリングする
+
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+_VerticalPodAutoscaler_ (VPA)を使用してワークロードを垂直方向に自動的にスケールできます。
+HPAと異なり、VPAはデフォルトでKubernetesに付属していませんが、[GitHubで](https://github.com/kubernetes/autoscaler/tree/9f87b78df0f1d6e142234bb32e8acbd71295585a/vertical-pod-autoscaler)見つかる別のプロジェクトです。
+
+インストールすることにより、管理されたレプリカのリソースを _どのように_ _いつ_ スケールするのかを定義するワークロードの{{< glossary_tooltip text="CustomResourceDefinitions" term_id="customresourcedefinition" >}}(CRDs)を作成できるようになります。
+
+{{< note >}}
+HPAが機能するにはクラスターに[Metrics Server](https://github.com/kubernetes-sigs/metrics-server)がインストールされている必要があります。
+{{< /note >}}
+
+現時点では、VPAは4つの異なるモードで動作できます:　
+
+{{< table caption="VPAの異なるモード" >}}
+モード | 説明
+:----|:-----------
+`Auto` | 現在、`Recreate`は将来インプレースアップデートに変更される可能性があります
+`Recreate` | VPAはPod作成時にリソースリクエストを割り当てるだけでなく、要求されたリソースが新しい推奨事項と大きく異なる場合にそれらを削除することによって既存のPod上でリソースリクエストを更新します
+`Initial` | VPAはPod作成時にリソースリクエストを割り当て、後から変更することはありません
+`Off` | VPAはPodのリソース要件を自動的に変更しません。推奨事項は計算され、VPAオブジェクトで検査できます
+{{< /table >}}
+
+#### インプレースリサイズの要件
+
+{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+
+{{< glossary_tooltip text="Pod" term_id="pod" >}}またはその{{< glossary_tooltip text="コンテナ" term_id="container" >}}を再起動**せずに**インプレースでワークロードをリサイズするには、Kubernetesバージョン1.27以降が必要です。
+さらに、`InPlaceVerticalScaling`フィーチャーゲートを有効にする必要があります。
+
+{{< feature-gate-description name="InPlacePodVerticalScaling" >}}
```

Comment:
In this line `{{< feature-gate-description name="InPlacePodVerticalScaling" >}}`, "feature-gate-description" shortcode is not being able to find the file for the “InPlacePodVerticalScaling” feature gate in the Japanese localization. To fix this, we need to localize the English file content in [content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md?plain=1) in Japanese or we can temporarily localize the description and add it directly here, then swap it when all the feature gate description files [(here)](https://github.com/kubernetes/website/tree/main/content/en/docs/reference/command-line-tools-reference/feature-gates) are localized. 
This is just a suggestion, and I will defer to the Japanese localization team for the best course of action.